config/axios.js

```diff
@@ -0,0 +1,19 @@
+const axios = require('axios');
+
+const axiosConfig = module.exports;
+
+axiosConfig.initAxios = () => {
+  const getResponse = response => {
+    const { data: responseData } = response;
+    return responseData;
+  };
+
+  const rejectResponse = error => {
+    const { message, response: { status } = { status: 500 } } = error;
```

Comment:
there's a file `app/errors.js` that can help you with error messages. You could use it.

Reply:
done